intergalactic-2/blocks.css

```diff
@@ -83,7 +83,6 @@ Description: Used to style Gutenberg Blocks.
 /* Captions */
 
 [class^="wp-block-"] figcaption {
-	color: inherit;	
```

Comment:
Happy to remove this, as we should default to remove block specific code

Reply:
👍